### PR TITLE
IWF-356: Add new fields to WorkflowOptions

### LINKED ISSUE #43: WorkflowAlreadyStarted Options
https://github.com/indeedeng/iwf-java-sdk/pull/258

### PR DESCRIPTION
**Pull request by lwolczynski**
Closes #43

### CODE REVIEW

File: iwf/tests/test_basic_workflow.py [hunk 1]
```diff
@@ -65,6 +65,11 @@ def get_workflow_states(self) -> StateSchema:
 def test_basic_workflow():
     wf_id = f"{inspect.currentframe().f_code.co_name}-{time.time_ns()}"
 
-    client.start_workflow(BasicWorkflow, wf_id, 100, "input")
+    client.start_workflow(
+        BasicWorkflow,
+        wf_id,
+        100,
+        "input",
```
Comment (longquanzheng) [inline iwf/tests/test_basic_workflow.py:72]: Did you want to add the test here? I feel like just passing the configOverride and no asserting is fine (we did the same Java SDK).

I also think it's better to test AlreadyStartedOptions.
Pass alreadyStartedOptions with a requestId here, and call the same start again, you would expect it return the same runID. 

Then pass a different requestId then you would expect an error thrown

**Comment by longquanzheng** [comment]
Left a comment. Looks great  otherwise!

File: iwf/tests/test_basic_workflow.py (hunk 1 above)
Comment (lwolczynski) [reply iwf/tests/test_basic_workflow.py:72]: I was using this test to do some manual checks. You're right, I'll add it here

Comment (lwolczynski) [reply iwf/tests/test_basic_workflow.py]: At the moment, this SDK doesn't return run_id at workflow start, so I can't assert that. Other that this, it's been implemented.

**Review by longquanzheng — APPROVED** [review]

**Pull request merged**